files.js

```diff
@@ -34,11 +34,18 @@ function calculateCRCAndWriteFile(buf, filename, cb) {
 function readFileAndCheckCRC(filename, cb) {
   readFile(filename, (err, buf) => {
     if (err) return cb(err)
+    if (buf.length < 16) return cb(Error('file too short'))
```

Comment:
Looks good. This function is used in all other load function so checking here is the right place 👍